app/domain/mail_relay/lists.rb

```diff
@@ -124,10 +124,18 @@ def prepare_bounced_message(list_address, sender_address)
       message.smtp_envelope_from = env_sender
     end
 
+    def domain_is_valid?(domain)
+        domain !~ /^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/
```

Comment:
Use 2 and not 4 Spaces for indentation
```suggestion
      domain !~ /^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$/
```